typescript/sdk/src/token/EvmERC20WarpModule.ts

```diff
@@ -0,0 +1,97 @@
+import { Address, ProtocolType, rootLogger } from '@hyperlane-xyz/utils';
+
+import { serializeContracts } from '../contracts/contracts.js';
+import { HyperlaneAddresses } from '../contracts/types.js';
+import {
+  HyperlaneModule,
+  HyperlaneModuleParams,
+} from '../core/AbstractHyperlaneModule.js';
+import { MultiProvider } from '../providers/MultiProvider.js';
+import { AnnotatedEV5Transaction } from '../providers/ProviderType.js';
+import { ChainNameOrId } from '../types.js';
+
+import { EvmERC20WarpRouteReader } from './EvmERC20WarpRouteReader.js';
+import { HypERC20Factories } from './contracts.js';
+import { HypERC20Deployer } from './deploy.js';
+import { TokenRouterConfig } from './schemas.js';
+
+export class EvmERC20WarpModule extends HyperlaneModule<
+  ProtocolType.Ethereum,
+  TokenRouterConfig,
+  HyperlaneAddresses<HypERC20Factories> & {
```

Comment:
afaik we dont actually serialize with all of these factory keys defined

Reply:
removing